packages/core/src/Common/Container.ts

```diff
@@ -38,6 +39,7 @@ export function createContainer(config: ConfigTypes.Config): Container {
   // bind default error provider
   container.bind(ErrorHandlerProvider, DefaultErrorHandlerProvider);
   container.bind(HttpServer);
+  container.bind(WebsocketServiceKey, WebsocketService);
```

Comment:
This should not be part of `core`. 

This should works like `auth`. The package have to be installed separately and inject to the IOC Container manually by the user like:
```ts
// playground/boot/Container.ts
export function useContainer(container: Container): void {

  // ...
  container.bind(WebsocketService); // The `key` I think is unnecessary

}